Formula/python3.rb

```diff
@@ -75,16 +75,16 @@ class Python3 < Formula
   # X11.
   patch :DATA if build.with? "tcl-tk"
 
-  def lib_cellar
+  def lib_cellar(xy)
     prefix/"Frameworks/Python.framework/Versions/#{xy}/lib/python#{xy}"
   end
 
-  def site_packages_cellar
-    lib_cellar/"site-packages"
+  def site_packages_cellar(xy)
+    lib_cellar(xy)/"site-packages"
   end
 
   # The HOMEBREW_PREFIX location of site-packages.
-  def site_packages
+  def site_packages(xy)
     HOMEBREW_PREFIX/"lib/python#{xy}/site-packages"
   end
```

Comment:
Let's just inline all these functions into variables in the needed methods e.g. `install`, `post_install`, etc.